test/testing-framework.js

```diff
@@ -0,0 +1,8 @@
+const assertEquals = (actualOutput, expectedOutput) => actualOutput === expectedOutput; // Needs to return true or false based on expected and actual outputs
+
+const assertNaN = actualOutput => isNaN(actualOutput);
+
+module.exports = {
+    assertEquals,
+    assertNaN,
```

Comment:
Next time I have a coding challenge, I will use Jasmine testing to thoroughly test my work instead of using a testing framework that has issues such as not being able to compare an array without changing it into a string.